dev.ts

```diff
@@ -1,2 +1,2 @@
-import { dev } from "./src/dev/mod.ts";
-export default dev;
```

Comment:
Why not leave the default export as dev? This seems like an unnecessary breaking change. Other than this, LGTM!